php/class-post-type.php

```diff
@@ -447,41 +451,82 @@ public function render_data_metabox( $post ) {
-
-			if ( '' === $value ) {
-				$preview = '<p><em>' . esc_html__( '(Empty string)', 'customize-snapshots' ) . '</em></p>';
-			} elseif ( is_string( $value ) || is_numeric( $value ) ) {
-				$preview = '<p>' . esc_html( $value ) . '</p>';
-			} elseif ( is_bool( $value ) ) {
-				$preview = '<p>' . wp_json_encode( $value ) . '</p>';
-			} else {
-				$preview = sprintf( '<pre class="pre">%s</pre>', esc_html( Customize_Snapshot_Manager::encode_json( $value ) ) );
+			if ( isset( $conflicts_settings[ $setting_id ] ) ) {
+				$setting_id_key = str_replace( '[', '\\[', $setting_id );
+				$setting_id_key = str_replace( ']', '\\]', $setting_id_key );
+				$title_text = sprintf( '%s ' . __( 'has potential conflicts (click to expand)', 'customize-snapshots' ), $setting_id );
+				echo '<a href="#TB_inline?width=600&height=550&inlineId=snapshot-' . esc_attr( $setting_id_key ) . '" class="dashicons dashicons-warning thickbox snapshot-thickbox" title="' . esc_attr( $title_text ) . '"></a>'; ?>
+				<div id="snapshot-<?php echo esc_attr( $setting_id ); ?>" style="display:none;">
+					<?php foreach ( $conflicts_settings[ $setting_id ] as $data ) { ?>
+						<details class="snapshot-conflict-details">
+							<summary>
+								<code>
+									<?php
+									echo esc_html( $data['uuid'] );
+									if ( ! empty( $data['name'] ) ) {
+										echo ' - ' . esc_html( $data['name'] );
+									}
+									?>
+									</code>
+								<a target="_blank" href="<?php echo esc_url( $data['edit_link'] ); ?>" class="dashicons dashicons-external"></a>
+							</summary>
+							<article class="snapshot-value">
+								<?php echo $this->get_printable_setting_value( $data['value'], $setting_id, $data['setting_param'], get_post( $data['id'] ) ); // WPCS: XSS ok. ?>
+							</article>
+						</details>
+					<?php } ?>
+				</div>
+				<?php
 			}
+			echo '</summary>';
 
-			/**
-			 * Filters the previewed value for a snapshot.
-			 *
-			 * @param string $preview HTML markup.
-			 * @param array  $context {
-			 *     Context.
-			 *
-			 *     @type mixed    $value          Value being previewed.
-			 *     @type string   $setting_id     Setting args, including value.
-			 *     @type array    $setting_params Setting args, including value.
-			 *     @type \WP_Post $post           Snapshot post.
-			 * }
-			 */
-			$preview = apply_filters( 'customize_snapshot_value_preview', $preview, compact( 'value', 'setting_id', 'setting_params', 'post' ) );
-
+			$preview = $this->get_printable_setting_value( $value, $setting_id, $setting_params, $post );
 			echo $preview; // WPCS: xss ok.
 			echo '</details>';
 			echo '</li>';
 		} // End foreach().
 		echo '</ul>';
 	}
 
+	/**
+	 * Get printable setting value
+	 *
+	 * @param mixed         $value Value to be printed.
+	 * @param string        $setting_id setting id.
+	 * @param array         $setting_params param raw array.
+	 * @param /WP_Post|null $post Post object of where setting belongs.
```

Comment:
A backslash should be used instead of a forward slash. Also, the param descriptions should be aligned vertically.